ports/fnafnes/FNAF NES.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+export PORT_32BIT="Y"
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+GAMEDIR="/$directory/ports/fnafnes"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```

Comment:
Wrong LD_LIBRARY_PATH Export.

Reply:
fixed

Reply:
you now need to rename `libs` to `libs.armhf` and check that gmloader still works :)